tower-http/src/cors.rs

```diff
@@ -4,7 +4,7 @@
 //!
 //! ```
 //! use http::{Request, Response, Method, header};
-//! use hyper::Body;
+//! use hypefut: futureBody;
```

Comment:
I guess this needs to be

```suggestion
//! use hyper::Body;
```

Reply:
Looks like rust-analyzer used some outdated spans when doing updating uses of a field I renamed 😅

Reply:
huh, that seems like maybe something that deserves an upstream bug report?